cookbook/core/control_flow/subworkflows.py

```diff
@@ -1,55 +1,45 @@
 """
-Subworkflows
+Subworkflows 
 ------------
 
-Launch plans were introduced in the Basics section of this book. Subworkflows are similar in that they allow users
-to kick off one workflow from inside another. What's the difference? Think of launch plans as passing by pointer and
-subworkflows as passing by value.
+Subworkflows are similar to :ref:`launch plans <Launch plans>` in that they allow users to kick off one workflow from inside another. 
 
-.. note::
+What's the difference? 
+Think of launch plans as pass by pointer and subworkflows as pass by value.
 
-    the real reason why subworkflows exist is because this is exactly how dynamic workflows are handled by flyte. So
-    instead of hiding the functionality, we expose the functionality at the user level. There are pros and cons of
-    using subworkflows as described below
+.. note::
 
-When you include a launch plan of workflow A inside workflow B, when B gets run, a new workflow execution,
-replete with a new workflow execution ID, a new Flyte UI link, will be run.
+    The reason why subworkflows exist is because this is exactly how dynamic workflows are handled by Flyte. So
+    instead of hiding this functionality, we expose it at the user level. There are pros and cons of
+    using subworkflows as described below.
 
-When you include workflow A as a subworkflow of workflow B, when B gets run, the entire workflow A graph is basically
+When you include Workflow A as a subworkflow of Workflow B, and when Workflow B is run, the entire graph of workflow A is simply
 copied into workflow B at the point where it is called.
 
-When should I use SubWorkflows?
+When Should I Use SubWorkflows?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-If you want to limit parallelism within a workflow and its launched subworkflows, Subworkflows provide a clean way
-to do that. This is because they execute within the same context of the parent workflow. Thus all nodes of a subworkflow
-will be constrained to the total constraint on the parent workflow
-
-When you use LaunchPlans within a workflow to launch an execution of a previously defined workflow, a new
-external execution is launched, with a separate execution ID and can be observed as a distinct entity in
-FlyteConsole/Flytectl etc. Moreover, the context is not shared, hence they may have separate parallelism constraints.
-We refer to these externalized invocations of a workflow using Launchplans from a parent workflow as
-``Child Workflows``.
-
-If your deployment is using multi-cluster setup, then Child workflows, may also allow you to spread the workload of a workflow
-potentially to multiple clusters.
+If you want to limit parallelism within a workflow and its launched subworkflows, subworkflows provide a clean way
```

Comment:
should it be sub-workflows or subworkflows? I see subworkflows everywhere. @samhita-alla

Reply:
"Subworkflows" here refers to "sub-workflows" literally, but not Flyte's "subworkflow". We can't say "launched workflow" because it isn't the same as "sub-workflow".